pages/common/uudecode.md

```diff
@@ -0,0 +1,8 @@
+# uudecode
+
+> Decode files encoded by `uuencode`.
+> More information: <https://manned.org/uudecode>.
+
+- Decode a file that was create with `uuencode` into the current directory:
```

Comment:
```suggestion
- Decode a file that was created with `uuencode` into the current directory:
```

Reply:
(oops, I prefer @marchersimon's proposal in [the comment below](https://github.com/tldr-pages/tldr/pull/5971/files#r633004204)) 😊